tests/conftest.py

```diff
@@ -191,6 +192,9 @@ def protect_hybrid_cloud_deletions(request):
             seen_models.add(fk_model)
             restrict_role(role="postgres_unprivileged", model=fk_model, revocation_type="DELETE")
 
+    restrict_role(role="postgres_unprivileged", model=OrganizationMember, revocation_type="CREATE")
```

Comment:
```suggestion
    restrict_role(role="postgres_unprivileged", model=OrganizationMember, revocation_type="INSERT")
```

Reply:
@corps `CREATE` is in invalid privilege option according to https://www.postgresql.org/docs/current/sql-revoke.html